drivers/InterruptManager.cpp

```diff
@@ -55,28 +56,33 @@ InterruptManager* InterruptManager::get() {
     return _instance;
 }
 
-InterruptManager::InterruptManager() {
+InterruptManager::InterruptManager()
+{
     // No mutex needed in constructor
-    memset(_chains, 0, NVIC_NUM_VECTORS * sizeof(CallChain*));
+    memset(_chains, 0, NVIC_NUM_VECTORS * sizeof(CallChain *));
 }
 
-void InterruptManager::destroy() {
+void InterruptManager::destroy()
+{
     // Not a good idea to call this unless NO interrupt at all
     // is under the control of the handler; otherwise, a system crash
     // is very likely to occur
     if (NULL != _instance) {
         delete _instance;
-        _instance = (InterruptManager*)NULL;
+        _instance = (InterruptManager *)NULL;
     }
 }
 
-InterruptManager::~InterruptManager() {
-    for(int i = 0; i < NVIC_NUM_VECTORS; i++)
-        if (NULL != _chains[i])
+InterruptManager::~InterruptManager()
+{
+    for (int i = 0; i < NVIC_NUM_VECTORS; i++)
+        if (NULL != _chains[i]) {
             delete _chains[i];
+        }
```

Comment:
I'm just going to assume this corresponds to the brace on line 75.

Reply:
Part of NULL check, line 79